tapfreighter/wallet.go

```diff
@@ -1387,56 +1350,9 @@ func (f *AssetWallet) FetchInternalKeyLocator(ctx context.Context,
 	return f.cfg.AddrBook.FetchInternalKeyLocator(ctx, rawKey)
 }
 
```

Comment:
Awesome to have all these checks moved into the LND `FundPsbt` call!

I reviewed that a bit and confirmed that dust checks and removal of dust change are implemented s.t. it would match the behavior we want here.